change/@fluentui-web-components-d74234e0-b903-4c51-af61-599648aee92d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat(web-components): enabling setting theme on an individual element and unsetting themes",
```

no need to provide scope, that's the purpose of change file driven release process, the change file is already scope.

```suggestion
  "comment": "feat: enabling setting theme on an individual element and unsetting themes",
```